travvy/src/components/OnboardingPage.vue

```diff
@@ -0,0 +1,154 @@
+<template>  
+  <div class="Onboarding">
+     <div class="container">
```

Comment:
im glad you broke the different section in this component into multiple div blocks, this will definitely organize our web page much better. this is something we should implement for all other pages to provide a great user experience.